src/defaults.js

```diff
@@ -834,7 +832,7 @@ var defaults = {
  * @see https://developer.mozilla.org/en-US/docs/Web/CSS/font
  */
 
-var textWidthCache = new LRUCache(2000);
+var fontMetrics = {};
 
```

Comment:
I don't see why the cache needed to be removed, but its not like we expect a million differents fonts on one grid